Remove npm background command (#6)
reboot-dev/resemble-hello-world: README.md

### Front end

> To run the front end example, in a separate terminal from the running `rsm dev` type:

Reviewer: ```suggestion
To run the front end, open a separate terminal and do:
```
